helm-chart-sources/logstream-workergroup/values.yaml

```diff
@@ -215,4 +215,8 @@ extraVolumeMounts: {}
 #       accessModes: [ "ReadWriteOnce" ]
 #       resources:
 #         requests:
-#           storage: 1Gi
\ No newline at end of file
+#           storage: 1Gi
+
+# Pods get 30s by default for orderly shutdown before they're killed but may
+# need a little more time in various scenarios. Set this to allow more time.
+# terminationGracePeriodSeconds: 60s
```

Comment:
Is there any reason not to just set this?  I'm guessing that unclean shutdowns have some negatives and the grace period isn't fully utilized unless needed, so people really won't pay unless they need the extra time.

Reply:
@benji-york I don't really know what an appropriate default should be so and my guess is that no matter what I would choose, someone would balk. So, I chose not to choose and just provide folk with the tool to use themself.

Reply:
Makes sense.